src/main.zig

```diff
@@ -259,9 +251,7 @@ pub fn main() !void {
                 std.log.info("Restarting fuzzer...", .{});
 
                 fuzzer.wait();
-                fuzzer.logPrincipal() catch {
-                    std.log.err("failed to log principal", .{});
-                };
+                try fuzzer.logPrincipal();
```

Comment:
if an error occurs here then the errdefer above will be calling `fuzzer.wait` again which is why I added the catch.

Reply:
oops, thanks!